Ideal-Media-Solutions-LLC/chadslist
# Fakedata

A method for quickly loading the local database with fake data


## Story / Task

see [Set up fake data](https://trello.com/c/kwwvCPUV)


## What this PR does?

This PR creates a DB set up process for seeding the database locally with fake data.

It includes a set up file and a faker file.


### Before 

We would have manually had to add data to the database.

### After

We can quickly populate the database with fake data

### How to test / reproduce

- `npm install` to make sure you have all of the dependencies installed
- `createdb chadslist` in the terminal if you haven't already created a database
- `backend/db/config` create a copy of the config.example.js and rename it to config.js
- In config.js put the username and password for accessing your local postgres server
- `node backend/db/setup/faker.js` In order to seed your database with fake data (There might be issues with this depending on how much you set up in the past but let me know)
- If having issue with generating fake data, try dropping the database chadslist in your Postgresql server, and re-create one: `drop database if exists chadslist;`, `create database chadslist;`


Thanks!
